app/helpers/value_helper.rb

```diff
@@ -24,11 +24,16 @@ def decimal?(val)
   end
 
   def property_to_s(val)
-    # assume all decimals are currency
-    if decimal?(val)
+    if decimal?(val) # assume all decimals are currency
       number_to_currency(val)
+    elsif val.is_a?(ActiveSupport::TimeWithZone)
+      I18n.l(val, format: :date)
+    elsif val == true
+      "Yes"
+    elsif val == false
+      "No"
     else
-      val.to_s
+      val
     end
```

Comment:
its funny, I never think to use case statements... guess I usually aim to have branching statements small enough that they wouldn't make sense...but I am interested: what about them is simpler? I dont fully understand why people use case statements vs `if` and `elsif`